tox.ini

```diff
@@ -6,6 +6,7 @@ envlist =
 isolated_build = true
 
 [testenv]
+alwayscopy = True
```

Comment:
What does this do?

Reply:
Ah sorry that's a change that has to be removed, I added this when tox wasn't considering the images in image_cache